tools/build-fs.sh

```diff
@@ -1,39 +1,41 @@
 #!/bin/bash
 # Copyright (c) 2018-2022 The MobileCoin Foundation
 
-# TODO: add a way to specify a different sigstruct directory
+# To use this script, run build-fs test or build-fs main
+# If no network is specified, or a different network is specified, the env's 
+#   version of the following variables are used
+#   - SGX_MODE
+#   - IAS_MODE
+#   - CONSENSUS_ENCLAVE_CSS
+#   - NAMESPACE (if no namespace is present, we use 'default')
+
+# TODO: 
 
 # Net can be main/test/local
-if [ $# -eq 0 ]; then
-NET="test"
-else
+if [ $# -gt 0 ]; then
 NET=$1
 fi
 
-if [ $NET == "local" ]; then
-echo "Local networks are not currently supported with this script"
-exit 1
-elif [ $NET == "test" ]; then
-echo "Building Full Service with testnet settings"
-NAMESPACE="test"
-export SGX_MODE=HW
-export IAS_MODE=PROD
-elif [ $NET == "main" ]; then
-NAMESPACE="prod"
-export SGX_MODE=HW
-export IAS_MODE=PROD
-else
-echo "Network specified is not valid. Try 'test' or 'main'. Default is test"
-exit 1
+if [ -z ${NET} ] || [ $NET -ne "test" ] || [ $NET -ne "main" ]; then
+echo "Using current environment's SGX_MODE, IAS_MODE, CONSENSUS_ENCLAVE_CSS "
+else 
+    if [ $NET == "test" ]; then
+    echo "Building Full Service with testnet settings"
+    NAMESPACE="test"
+    export SGX_MODE=HW
+    export IAS_MODE=PROD
+    elif [ $NET == "main" ]; then
+    NAMESPACE="prod"
+    export SGX_MODE=HW
+    export IAS_MODE=PROD
+    fi 
+    
+    WORK_DIR="$HOME/.mobilecoin/${NET}"
```

Comment:
What will happen here if NET is not set?

Reply:
IT appears that we can get here with `NET` not being set to anything, which would just set `WORK_DIR` to `$HOME/.mobilecoin/`

Reply:
Ah, my mistake. I misread the `if` statement.
Looks like it can be simplified to just `[ $NET == "test" ] || [ $NET == "main" ]` since that already implies `$NET` is not empty

Reply:
This should be mentioned in a comment then.
Also, it would be helpful for me if conversations are not marked as resolved until I have a chance to look at them, otherwise I can't tell what really was addressed and what still has a followup (such as documenting this behavior in a comment). Thanks!

Reply:
Unrelated, I believe that if this is changed to `"$NET" == "test"` then there won't be need for the `-z` test...

Reply:
Thank you!